webui/src/pages/repositories/repository/fileRenderers/index.tsx

```diff
@@ -178,7 +179,14 @@ export function guessType(contentType: string | null, fileExtension: string | nu
 }
 
 export const ObjectRenderer: FC<RendererComponent> = (props: RendererComponent) => {
-   const fileType = guessType(props.contentType, props.fileExtension)
+    const pluginManager = usePluginManager();
+
+    const customRenderer = pluginManager.customObjectRenderers.get(props.contentType, props.fileExtension)
```

Comment:
The signature of get is for contentType and fileExtension is string, while their type is `string | null` 
FWIU you should change the signature or pass undefined in case of null (i.e `(props.contentType ?? undefined, props.fileExtension ?? undefined)`

Reply:
Yeah, apparently even IntelliJ noted about this...

Updated - 
It required some more null/undefined changes in other files...